src/theme/darkTheme.ts

```diff
@@ -23,4 +23,5 @@ export const darkTheme = {
     ratingGreat: '#BD2430',
     ratingAverage: '#6dc8b7',
     ratingNotRated: '#CCC',
+    title: '#000',
```

Comment:
Místo tohohle bych použil už existující barvu, třeba textOnLightDark. Čistě černou tam teď nikde nepoužíváme.

Reply:
Jo, to je dobrý nápad, fixed. Thx.